app/src/main/java/com/infomaniak/mail/ui/main/menu/MenuDrawerFragment.kt

```diff
@@ -165,7 +165,9 @@ class MenuDrawerFragment : MenuFoldersFragment(), MailboxListFragment {
         }
 
         appVersionName.setOnClickListener {
-            (requireActivity() as MainActivity).onEasterEggConfettiClicked(matomoValue = "MenuDrawer")
+            (activity as? MainActivity)?.binding?.easterEggConfettiContainer?.let {
```

Comment:
make a getConfettiContainer() function in mainActivity, it would reduce binding access and also it is used 2 times so factorizing it would be good